src/expandableCalendar/index.tsx

```diff
@@ -583,6 +583,7 @@ const ExpandableCalendar = (props: ExpandableCalendarProps) => {
         numberOfDays={numberOfDays}
         headerStyle={_headerStyle}
         timelineLeftInset={timelineLeftInset}
+        context={useContext(Context)}
```

Comment:
Why are you passing context like this and not using Context provider? 
You can create a new Context just for this behavior

Reply:
Why create a new context for this?

Reply:
You expect the user to tell if it's controlled/uncontrolled? 

Isn't it inherited by the components the user use? 
I mean if they use Expandable, you can tell it's uncontrolled and the other way around. 

It's a little weird to have the user decide on this, it's an internal implementation detail

Reply:
The user does NOT decide that at all. It is exactly how you described it - if the user uses ExpandableCalendar it is controlled since the Expandable is passing the 'context' prop. Other components do not thus they are uncontrolled. This is why the 'context' prop is private and not exposed to the user.

Reply:
So am I missing something? Where in the code do you pass the `isControlled` in the context?

Reply:
`isControlled` is just a local const (I named it such to indicate that this is controlled/uncontrolled component. I could name it 'hasContext'...), if there is 'context' I pass 'disableDaySelection' to the 'getState()' function.